### PR TITLE
Unbreak the KUAL extension

### PR DESCRIPTION
**Pull request by NiLuJe**
Nasty symlink shenanigans meant we weren't actually shipping a required script...

Also, unclutter it while I'm here ;).

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/5715)

### CODE REVIEW

**Comment by NiLuJe** [comment]
(Waiting on CI to double-check that I haven't borked the symlink).

File: platform/kindle/extensions/koreader/bin/koreader-ext.sh [hunk 1]
```diff
@@ -93,55 +93,6 @@ install_koreader() {
     update_koreader "clean"
 }
 
-# Handle cre's settings...
```
Comment (Frenzie) [inline platform/kindle/extensions/koreader/bin/koreader-ext.sh]: What even is all this? Some vestigial thing from 2011? >_>

Comment (NiLuJe) [reply platform/kindle/extensions/koreader/bin/koreader-ext.sh]: Yeah, reaaaaaallly old stuff ^^.

**Comment by NiLuJe** [comment]
@Frenzie: Any idea why the CI is only now complaining about not having ragel to build HB from git?

**Comment by Frenzie** [comment]
Some failed download? That's… pretty weird.

**Comment by NiLuJe** [comment]
No, it tried to regenerate some headers (which I think actually *are* shipped in the git repo, so, some kind of timestamp mishap?).

**Comment by Frenzie** [comment]
Just rerun it?

**Comment by NiLuJe** [comment]
Just remembered that front's CI only does an emulator build, so, running a local kindle build to double-check ;p.

**Comment by NiLuJe** [comment]
```
  adding: extensions/koreader/bin/libkohelper.sh (deflated 53%)
```

Wheee!

**Pull request merged**